src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2356,11 +2357,13 @@ protected void Log(string msg, WarningLevel severity)
         /// the data is split into two parts, model and view. This method sets the view information.
         /// </summary>
         /// <param name="workspaceViewInfo">The extra view information from the workspace to update the model with.</param>
-        public void UpdateWithExtraWorkspaceViewInfo(ExtraWorkspaceViewInfo workspaceViewInfo)
+        /// <param name="centerNodes">Should Nodes be centered at their locations.</param>
+        public void UpdateWithExtraWorkspaceViewInfo(ExtraWorkspaceViewInfo workspaceViewInfo, double offsetX = 0.0, double offsetY = 0.0)
```

Comment:
Adding default parameters will cause be a API breaking change. 
https://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net

Reply:
Created an overloaded version of the method taking the parameters.